crates/invoker-impl/src/invocation_task/service_protocol_runner.rs

```diff
@@ -201,8 +202,10 @@ where
         deployment_metadata: DeploymentMetadata,
         service_protocol_version: ServiceProtocolVersion,
         invocation_id: &InvocationId,
-    ) -> (Sender, Request<Body>) {
-        let (http_stream_tx, req_body) = Body::channel();
+    ) -> (InvokerRequestStreamSender, Request<InvokerBodyStream>) {
+        // Just an arbitrary buffering size
+        let (http_stream_tx, http_stream_rx) = mpsc::channel(10);
```

Comment:
Interesting, there was no buffering whatsoever... https://github.com/hyperium/hyper/blob/618a18580ae14776c05ab6789a82b5c43281fe8b/src/body/body.rs#L145

Let's keep 10, I'll investigate whether it could be just `1` or `0`